### PR TITLE
API Provider - No Namespace Selected Page

### PR DESCRIPTION
**Pull request by jTendeck**
https://api-services-portal-feature-api-provider-no-ns.apps.silver.devops.gov.bc.ca/

Once logged in and while no namespace is selected, click `For API Providers` to visit page (or visit `/manager/namespaces`)

### CODE REVIEW

File: src/nextapp/components/namespace-menu/namespace-menu.tsx [hunk 1]
```diff
@@ -16,15 +16,20 @@ import { useQueryClient } from 'react-query';
...
 
-const NamespaceMenu: React.FC<NamespaceMenuProps> = ({ user }) => {
+const NamespaceMenu: React.FC<NamespaceMenuProps> = ({
+  user,
+  variant,
+  buttonMessage,
+}) => {
   const client = useQueryClient();
   const toast = useToast();
   const newNamespaceDisclosure = useDisclosure();
```
Comment (Jonesy) [inline src/nextapp/components/namespace-menu/namespace-menu.tsx:35]: Very minor, but if there is only one `useDisclosure` hook being used in a component just destruct it to save some typing when you use the props, ie `const { isOpen, onClose, onOpen } = useDisclosure();`

File: src/nextapp/pages/manager/namespaces/index.tsx [hunk 2]
```diff
@@ -96,14 +97,15 @@ const secondaryActions = [
...
+
 const NamespacesPage: React.FC = () => {
   const { user } = useAuth();
   const hasNamespace = !!user?.namespace;
   const router = useRouter();
   const toast = useToast();
   const mutate = useApiMutation(mutation);
   const client = useQueryClient();
-  const { isOpen, onClose, onOpen } = useDisclosure();
+  const newNamespaceDisclosure = useDisclosure();
```
Comment (Jonesy) [inline src/nextapp/pages/manager/namespaces/index.tsx:108]: Same thing here, I would just undo this change, single use destructing is fine.

**Review by Jonesy — APPROVED** [review]
Looks good, there are few minor stylistic things but no blockers for the merge.

**Comment by ikethecoder** [comment]
@jTendeck just a few conflicts to resolve and if you want to do Josh's small tweaks.. I will then merge in to include with Production release
